server/src/main/java/io/spine/server/command/model/CommandHandlerMethod.java

```diff
@@ -161,4 +135,42 @@ protected boolean verifyReturnType(Method method) {
             return result;
         }
     }
+
+    /**
+     * The result of a command handler method execution.
+     */
+    public static final class Result extends EventsResult {
+
+        private Result(EventProducer producer, Object rawMethodResult) {
+            super(producer, rawMethodResult);
+            List<Message> eventMessages = toMessages(rawMethodResult);
+            List<Message> withoutEmpty = filterEmpty(eventMessages);
+            checkResultNonEmpty(withoutEmpty, rawMethodResult, producer);
+            setMessages(withoutEmpty);
```

Comment:
`withoutEmtpy` is confusing in 2 lines above. They read like "check result non-empty in without empty" and "set messages to without empty".
Can we have it `filteredMessages`, `filtered` or something else? 
It would read like
```java
checkResultNonEmpty(filteredMessages, rawMethodResult, producer);
setMessages(filteredMessages);
```